packages/gated-content/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "@lens-protocol/gated-content",
+  "version": "0.0.1",
+  "description": "Token-gated content for the Lens Protocol",
+  "main": "dist/lens-protocol-gated-content.cjs.js",
+  "module": "dist/lens-protocol-gated-content.esm.js",
+  "exports": {
+    ".": {
+      "module": "./dist/lens-protocol-gated-content.esm.js",
+      "default": "./dist/lens-protocol-gated-content.cjs.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "repository": {
+    "directory": "packages/gated-content",
+    "type": "git",
+    "url": "git://github.com/lens-protocol/lens-sdk.git"
+  },
+  "sideEffects": false,
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "preconstruct build",
+    "eslint:fix": "pnpm run eslint --fix",
+    "eslint": "eslint src",
+    "lint": "pnpm run prettier && pnpm run eslint && pnpm run tsc",
+    "lint:fix": "pnpm run prettier:fix && pnpm run eslint:fix && pnpm run tsc",
+    "prettier:fix": "prettier --write .",
+    "prettier": "prettier --check .",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "tsc": "tsc --noEmit"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "@ethersproject/address": "^5.7.0",
```

Comment:
curious, given that we also install full `ethers`, does that mean it's possible to end up with duplicated code when both `@ethersproject/address` and `ethers` address have the same code?